src/status_im/contexts/wallet/edit_account/view.cljs

```diff
@@ -62,13 +62,14 @@
             button-disabled?        (or (nil? @edited-account-name)
                                         (= name @edited-account-name))]
         [create-or-edit-account/view
-         {:page-nav-right-side [{:icon-name :i/delete
-                                 :on-press
-                                 (fn []
-                                   (rf/dispatch [:show-bottom-sheet
-                                                 {:content
-                                                  (fn []
-                                                    [remove-account/view])}]))}]
+         {:page-nav-right-side [(when-not default-account?
```

Comment:
there is one more issue in this feature epic - https://github.com/status-im/status-mobile/issues/18575

can we hide these on-presses behind a feature-flag for now? and then we will review all of this in go?

Reply:
see https://github.com/status-im/status-mobile/pull/18569/files#diff-f121c6f060282c0f68513d7962d77967ef0cd9b5312c19f6b59304fed19c0108 for an example of what I mean